web-client/public/js/constants.js

```diff
@@ -37,3 +37,7 @@ export const WEIGHTS_HIDE_SIDE             = "#weightsNeverSide";
 export const WEIGHTS_SHOW_MOUSE_OVER_CLASS = ".weightsMouseOver";
 export const WEIGHTS_SHOW_ALWAYS_CLASS     = ".weightsAlways";
 export const WEIGHTS_HIDE_CLASS            = ".weightsNever";
+
+export const showWeightsMouseover = "showWeightsMouseover";
+export const showAllWeights = "showAllWeights";
+export const hideAllWeights = "hideAllWeights";
```

Comment:
Note our convention with constants: all caps, underscore to separate words. It isn't a necessity but it makes constants easier to spot when looking at code.